src/main/webapp/scripts/map.js

```diff
@@ -88,17 +88,20 @@ function initMap() {
   const origin = {
     lat: 51.533364, 
     lng: -0.125777
-  };
   const map = new google.maps.Map(document.getElementById("map"), {
     center: origin,
     zoom: 15
   });
+  
+  /* Get all stored markers */
+  fetchMarkers();
```

Comment:
Please add `map` as a parameter her as it's no longer in the global scope, same for `placeMarkerAndPanTo` and `addHardCodedMarkers`